src/main.ts

```diff
@@ -157,3 +165,7 @@ app.on('activate', () => {
     createWindow()
   }
 })
+
+app.on('ready', async () => {
+  global.pastelDB = await createDatabase()
```

Comment:
Please handle the exception with try/catch

Reply:
Good advise, Thank you.